package.json

```diff
@@ -99,7 +99,7 @@
     "@rollup/plugin-json": "^4.0.1",
     "@rollup/plugin-node-resolve": "^6.0.0",
     "@rollup/plugin-replace": "^2.3.0",
-    "@rollup/plugin-sucrase": "^3.0.0",
+    "@rollup/plugin-sucrase": "^3.1.0",
```

Comment:
If you rebase against `master` you should be able to drop your changes to this file and `package-lock.json`. I submitted those as a separate PR in https://github.com/sveltejs/svelte/pull/5758 because that was a big enough change by itself that it took some review. Hopefully that'll make review of this one a bit easier by making this one a bit smaller